### DISCUSSION TITLE
How can I get "brew tests" to work?

### DISCUSSION THREAD

**Discussion by denismm** [Homebrew/discussions#1541] [Tap maintenance and Homebrew development] [answered]
I'm attempting to fix the ghostscript formula and following the instructions at https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request but "brew tests" fails.  Output attached.  It's not at obvious to me what could be causing these errors.

The attached log is from running brew tests from fish but I tried under bash and still see the same errors.

[testfailures.txt](https://github.com/Homebrew/discussions/files/6542673/testfailures.txt)

**Comment by carlocab** [comment, marked as answer]
If you're working on a PR that affects only the `ghostscript` formula, you need to run
```
brew test ghostscript
brew audit --strict ghostscript
```
You need to run `brew tests` only if you're making changes to `brew` itself (i.e. the code that lives in `$(brew --repo)/Library/Homebrew`).

The docs could probably be improved here. Thanks for following them so carefully, though.

**Reply by denismm** [reply]
Thank you for the clarification! :)
